mitosheet/src/components/taskpanes/AITransformation/AITransformationResultSection.tsx

```diff
@@ -117,6 +121,43 @@ const AITransformationResultSection = (props: AITransformationResultSectionProps
                     No changes
                 </p>
             }
+            {(result.created_dataframe_names.length > 0 || Object.entries(result.modified_dataframes_recons).length > 0 || result.deleted_dataframe_names.length > 0) &&
+                props.isMostRecentResult &&
+                <Row 
+                    justify="space-between" 
+                    onClick={() => {
+                        props.setUIState(prevUIState => {
+
+                            if (prevUIState.dataRecon !== undefined) {
+                                return {
+                                    ...prevUIState, 
+                                    dataRecon: undefined
+                                }
+                            }
```

Comment:
Why does this make sense? Maybe a comment here to clarify

Reply:
Comment added -- if the user clicks the eye when the recon is already displayed, then we want to turn off the recon.